### ISSUE TITLE
'Converter' object has no attribute '_add_bw_entry_to_entries_dict'. 

### ISSUE THREAD

**Issue by shiyunjin** [jampe/kp2bw#39]
**Describe the bug**
https://github.com/jampe/kp2bw/pull/29 A syntax error occurred after being merged, `_add_bw_entry_to_entries_dict` was rename `_add_bw_entry_to_entires_dict`

**Provide verbose program output**
```
  File "C:\kp2bw-master\.env\kp2bw\Lib\site-packages\kp2bw\convert.py", line 180, in _load_keepass_data
    self._add_bw_entry_to_entries_dict(entry, custom_protected)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Converter' object has no attribute '_add_bw_entry_to_entries_dict'. Did you mean: '_add_bw_entry_to_entires_dict'?
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Python version: 3.11.1
 - Bitwarden cli version: 2023.1.0

**Additional context**

**Comment by shiyunjin** [comment]
PR fix: https://github.com/jampe/kp2bw/pull/40

**Comment by jampe** [comment]
fixed in #40 thanks @shiyunjin
